### PR TITLE
correct some typos

### PR DESCRIPTION
**Pull request by tenzap**

### CODE REVIEW

**Comment by gvanem** [comment]
I believe **ADC overload** is the correct term.

**Comment by tenzap** [comment]
Oh, really ? Actually I was wondering. What does ADC stand for ?

**Comment by gvanem** [comment]
> What does ADC stand for ?

*Analogue to Digital Converter*; the basis of all SDRs. 
Lots of info here: https://www.rtl-sdr.com/tag/overload/

File: src/input/rtl_tcp.cpp [hunk 1]
```diff
@@ -546,8 +546,7 @@ void CRTL_TCP_Client::agcTimer(void)
         }
         else { // AGC is off or unknown tuner
             if (minAmplitude == 0 || maxAmplitude == 255) {
-                std::string text = QT_TRANSLATE_NOOP("CRadioController", "ADC overload."
-                    " Maybe you are using a too high gain.");
+                std::string text = QT_TRANSLATE_NOOP("CRadioController", "ADC overload. Maybe you are using a too high gain.");
```
Comment (marcusmueller) [inline src/input/rtl_tcp.cpp:549]: Even if this just seems to undo consistent line breaking (which I'd argue is undesirable in absence of an unambiguous guideline): 

This seems unidiomatic. I believe a native user would say

"Your gain might be [set] too high."

So, if you change this line, then please make it sound more "round".

File: src/welle-gui/QML/settingpages/RTLSDRSettings.qml [hunk 2]
```diff
@@ -18,7 +18,7 @@ SettingSection {
     WSwitch {
         id: enableBiasTee
         Layout.fillWidth: true
-        text: qsTr("Enable bias tee (not from all dongles supported)")
+        text: qsTr("Enable bias tee (not supported by all dongles)")
```
Comment (marcusmueller) [inline src/welle-gui/QML/settingpages/RTLSDRSettings.qml:21]: End sentence in "."

File: src/welle-gui/android_rtl_sdr.cpp [hunk 3]
```diff
@@ -136,7 +136,7 @@ void CAndroid_RTL_SDR::setOpenInstallDialog()
     // Init pop up
     QAndroidJniObject okButtonText = QAndroidJniObject::fromString(QT_TR_NOOP("OK"));
     QAndroidJniObject cancelButtonText = QAndroidJniObject::fromString(QT_TR_NOOP("Cancel"));
-    QAndroidJniObject message = QAndroidJniObject::fromString(QT_TR_NOOP("Android RTL-SDR driver is not installed. Do you would like to install it? After install start welle.io again."));
+    QAndroidJniObject message = QAndroidJniObject::fromString(QT_TR_NOOP("Android RTL-SDR driver is not installed. Would you like to install it? After install start welle.io again."));
```
Comment (marcusmueller) [inline src/welle-gui/android_rtl_sdr.cpp:139]:
```
- after install
+ after installation
```

**Review by marcusmueller — CHANGES_REQUESTED** [review]

File: src/welle-gui/android_rtl_sdr.cpp (hunk 3 above)
Comment (tenzap) [reply src/welle-gui/android_rtl_sdr.cpp]: changed

File: src/welle-gui/QML/settingpages/RTLSDRSettings.qml (hunk 2 above)
Comment (marcusmueller) [reply src/welle-gui/QML/settingpages/RTLSDRSettings.qml:21]: good point, I guess not. But seeing the position of said button, the text seems to be too long (not your fault).

**Review by AlbrechtL — APPROVED** [review]

**Pull request merged**